src/command/cmd_funcs.h

```diff
@@ -128,6 +128,7 @@ gboolean cmd_status_get(ProfWin* window, const char* const command, gchar** args
 gboolean cmd_status_set(ProfWin* window, const char* const command, gchar** args);
 gboolean cmd_sub(ProfWin* window, const char* const command, gchar** args);
 gboolean cmd_theme(ProfWin* window, const char* const command, gchar** args);
+gboolean cmd_tiny(ProfWin* window, const char* const command, gchar** args);
```

Comment:
Seems this is a mistake to introduce old tiny?

Reply:
Haha yes! Must be a bad rebase, I gues!